src/Naming/Base64Namer.php

```diff
@@ -17,7 +17,7 @@ class Base64Namer implements NamerInterface, ConfigurableInterface
     protected const ALPHABET = '0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ-_';
 
     /** @var int Length of the resulting name. 10 can be decoded to a 64-bit integer. */
-    protected $length = 10;
```

Comment:
This is a protected property of an extensible class, altering it is a BC break. If you want this to be merged soon, you need to revert this change.

Reply:
@garak fixed